src/Entity/PartyEvent.php

```diff
@@ -20,34 +25,37 @@
         new Post(),
         new Put(),
         new Delete(),
-    ]
+    ],
+    formats: ['json'],
+    collectDenormalizationErrors: true
 )]
+#[ApiFilter(filterClass: SearchFilter::class, properties: ['name' => 'partial'])]
```

Comment:
### ❗ Issue: Pls add posibility to search `PartyEvents` by `user_uid`

We need it to load all entities of current user